### PR TITLE
show points for users on feed

### PR DESCRIPTION
**Pull request by princessguerrero**
show points as percent matched on user feed, rounded points to the nearest tenths, edit sorting algorithm to sort descending.

### CODE REVIEW

File: front-end/src/LoggedInUser/FEED/MatchedBuddies.jsx [hunk 1]
```diff
@@ -221,11 +221,10 @@ class MatchedBuddies extends Component {
 	}
 
 	sortByPoints = () => {
-		let sorted = this.state.allUsersTrips.sort((a, b) => a.points - b.points)
-console.log('we is getting osrted', sorted)
+		let sorted = this.state.allUsersTrips.sort((a, b) => b.points - a.points)
+console.log('we is getting sorted', sorted)
```
Comment (michellelala) [inline front-end/src/LoggedInUser/FEED/MatchedBuddies.jsx]: please delete this console log (haha)

File: front-end/src/LoggedInUser/FEED/UserProfileCards.jsx [hunk 2]
```diff
@@ -7,16 +7,19 @@ import AllBuddies from "./AllBuddies";
 
 const UserProfileCards = ({ allUsers, bffs }) => {
   allUsers.map(user=> {
+    console.log("user profile ", user)
```
Comment (michellelala) [inline front-end/src/LoggedInUser/FEED/UserProfileCards.jsx]: please delete this console log

File: front-end/src/LoggedInUser/FEED/UserProfileCards.jsx [hunk 3]
```diff
@@ -7,16 +7,19 @@ import AllBuddies from "./AllBuddies";
...
   
   return (
     <div className="cardHolder">
       {allUsers.map(user => {
         return (
           <div className="card">
             <img className="pic" src={user.pic} alt="profile picture" />
-            <div className="name-age"> {user.first_name}, {user.age}, {user.my_location}</div>
+            <div className="name-age"> {user.first_name}, {user.age}, {user.my_location} </div>
+        <div>{user.points ? <div>you are {user.points}% matched</div> : <div>you are not matched </div>}</div>
```
Comment (michellelala) [inline front-end/src/LoggedInUser/FEED/UserProfileCards.jsx]: Could we change "You are {user.points}% matched" to "{user.points}% match"?
Additionally, "You are not matched" to "0% match"

File: front-end/src/LoggedInUser/FEED/MatchedBuddies.jsx (hunk 1 above)
Comment (princessguerrero) [reply front-end/src/LoggedInUser/FEED/MatchedBuddies.jsx]: ok deleted it haha

File: front-end/src/LoggedInUser/FEED/UserProfileCards.jsx (hunk 2 above)
Comment (princessguerrero) [reply front-end/src/LoggedInUser/FEED/UserProfileCards.jsx]: deleted this too

File: front-end/src/LoggedInUser/FEED/UserProfileCards.jsx (hunk 3 above)
Comment (princessguerrero) [reply front-end/src/LoggedInUser/FEED/UserProfileCards.jsx]: changed.  Thanks!

**Pull request merged**